client/my-sites/site-settings/disconnect-site/why/disconnect-reasons-suggestion.jsx

```diff
@@ -0,0 +1,172 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from 'components/select-dropdown';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+class DisconnectReasonsSuggestion extends Component {
+	constructor( props ) {
+		super( props );
+		const initialState = {
+			childSelected: 'default',
+			compactButtons: false,
+		};
+		this.state = initialState;
+	}
+	componentWillReceiveProps() {
+		const state = {
+			childSelected: 'default',
+		};
+		this.state = state;
+	}
+
+	_logReasonFollowUp = ( option ) => {
+		this.setState( {
+			childSelected: option.value,
+		} );
+	}
+	setOptions = ( optionValue ) => {
+		let options = [];
+		switch ( optionValue ) {
+			case 'missingFeature':
+				options = [
+					{ value: 'default', label: 'Backup' },
+					{ value: 'missingFeature_followup_themes', label: 'Themes' },
+					{ value: 'missingFeature_followup_other', label: 'Love :)' },
+				];
+				break;
+
+			case 'reason2':
+				options = [
+				{ value: 'default', label: 'sth1' },
+				{ value: 'reason2_followup_sth2', label: 'sth2' },
```

Comment:
I'd assume these labels and values are still in progress? They will need to be updated accordingly.